esp/src/eclwatch/HPCCPlatformWidget.js

```diff
@@ -415,7 +416,18 @@ define([
         _onAboutClose: function (evt) {
             this.aboutDialog.hide();
         },
-
+		
+        _onLogout: function (evt) {
+            xhr("esp/logout", {
```

Comment:
@kunalaswani - does it work when ECL Watch is loaded with this alternative URL:  http://x.x.x.x:8010/esp/files/stub.htm?Widget=HPCCPlatformWidget